dockers/tpu-tests/tpu_test_cases.jsonnet

```diff
@@ -33,6 +33,7 @@ local tputests = base.BaseTest {
       echo $KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS
       export XRT_TPU_CONFIG="tpu_worker;0;${KUBE_GOOGLE_CLOUD_TPU_ENDPOINTS:7}"
       coverage run --source=pytorch_lightning -m pytest -v --capture=no \
+          tests/plugins/test_tpu_spawn.py \
```

Comment:
ok, this can be easily generated on the fly...
cc: @awaelchli

Reply:
Yes, started to work on this here: #11098